rclcpp_components/include/rclcpp_components/register_node_macro.hpp

```diff
@@ -32,7 +32,7 @@
  */
 #define RCLCPP_COMPONENTS_REGISTER_NODE(NodeClass) \
   CLASS_LOADER_REGISTER_CLASS( \
-    rclcpp_components::NodeFactoryTemplate<NodeClass>, \
-    rclcpp_components::NodeFactory)
+  rclcpp_components::NodeFactoryTemplate<NodeClass>, \
+  rclcpp_components::NodeFactory)
```

Comment:
@rotu losing indentation here is unfortunate.

Reply:
It is unfortunate. https://github.com/ament/ament_lint/pull/174#issuecomment-582688531

I don't understand why, and I think it's an uncrustify bug.